README.md

```diff
@@ -1,162 +1,74 @@
+https://github.com/mozilla/Addon-Tests/contributors
 
-[mit]: http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-189-a-gentle-introduction-to-programming-using-python-january-iap-2011/
-[dive]: http://www.diveintopython.net/toc/index.html
-[webqa]: http://quality.mozilla.org/teams/web-qa/
-[firefox]: http://quality.mozilla.org/teams/desktop-firefox/
 [webdriver]: http://seleniumhq.org/docs/03_webdriver.html
+[runningtests]: https://developer.mozilla.org/en-US/docs/Mozilla/QA/Running_Web_QA_automated_tests
 
 Questions are always welcome
 ----------------------------
-While we take pains to keep our documentation updated, the best source of information is those of us who work on the project.  Don't be afraid to join us in irc.mozilla.org [#mozwebqa][mozwebqa] to ask questions about our Selenium tests.  Mozilla also hosts the [#mozillians][mozillans] chat room to answer your general questions about contributing to Mozilla.
+While we take pains to keep our documentation updated, the best source of information is those of us who work on the project.  Don't be afraid to join us in irc.mozilla.org [#mozwebqa][mozwebqa] to ask questions about our Selenium tests. We also have a [mailing list][mailing_list] available that you are welcome to join and post to.
 
-[mozwebqa]:http://02.chat.mibbit.com/?server=irc.mozilla.org&channel=#mozwebqa
-[mozillians]:http://02.chat.mibbit.com/?server=irc.mozilla.org&channel=#mozillians
+[mozwebqa]:http://widget01.mibbit.com/?settings=1b10107157e79b08f2bf99a11f521973&server=irc.mozilla.org&channel=%23mozwebqa
+[mailing_list]:https://mail.mozilla.org/listinfo/mozwebqa
 
-How to Set up and Build AMO Tests Locally
+How to run the tests locally
 -----------------------------------------
-This repository contains Selenium tests used to test the website addons.mozilla.org.
-
-Mozilla maintains a guide to running Automated tests on our MDN website:
-
-https://developer.mozilla.org/en-US/docs/Mozilla/QA/Running_Web_QA_automated_tests
-
-###You will need to install the following:
+We maintain a [detailed guide][runningtests] to running our automated tests which can be found on the MDN website. If you want to get started quickly, you can try following the steps in our quick start instructions below:
 
-#### Git
-If you have cloned this project already then you can skip this!
-GitHub has excellent guides for [Windows][GitWin], [MacOSX][GitMacOSX] and [Linux][GitLinux].
-[GitWin]: http://help.github.com/win-set-up-git/
-[GitMacOSX]: http://help.github.com/mac-set-up-git/
-[GitLinux]: http://help.github.com/linux-set-up-git/
+###Clone the repo
+If you have cloned this project already then you can skip this, otherwise you'll need to clone this repo using Git. If you do not know how to clone a GitHub repository, check out this [help page] (https://help.github.com/articles/cloning-a-repository/) from GitHub.
```

Comment:
@davehunt We discussed the question of forking in the meeting and my recollection was that we decided that is something we could discuss with them if they got to the point of wanting to help maintain the tests. That is why I simply left it out.